calendars/gregorian-star-trek-variants.json

```diff
@@ -184,14 +184,58 @@
     },
     "federation-standard": {
       "name": "Federation Standard Calendar",
-      "description": "United Federation of Planets standard calendar",
+      "description": "United Federation of Planets standard calendar with comprehensive stardate formatting",
+      "default": false,
       "config": {
         "yearOffset": 2161
       },
       "overrides": {
         "year": {
           "prefix": "",
-          "suffix": " F.S."
+          "suffix": ""
+        },
+        "dateFormats": {
```

Comment:
Is the `dateFormats` property supported in variants? Is there a test demonstrating this?